spec/ios_lint_localizations_spec.rb

```diff
@@ -205,5 +209,16 @@ def run_l10n_linter_test(data_file:, check_duplicate_keys: nil)
 
       expect(result).to eq({ 'fr' => ['`key3` expected placeholders for [Int] but found [] instead.'] })
     end
+
+    it 'fails on extra strings in translations by default' do
+      run_l10n_linter_test(data_file: 'extra-strings-in-translations-error')
+    end
+
+    it 'do not report extra strings in translations when fail_on_extra_strings is false' do
+      run_l10n_linter_test(
+        data_file: 'extra-strings-in-translations-no-error',
+        fail_on_extra_strings: false
+      )
+    end
```

Comment:
Worth adding a test for the positive value, just in case?

```suggestion
    end
    
    it 'does report extra strings in translations when fail_on_extra_strings is true' do
      run_l10n_linter_test(
        data_file: 'extra-strings-in-translations-error',
        fail_on_extra_strings: true
      )
    end
```